proto/adaptors/balancer/balancer_pool.proto

```diff
@@ -158,6 +181,82 @@ message BalancerPoolAdaptorV1 {
+            // Represents function calls to the CompoundCTokenAdaptor V2
+            CompoundCTokenAdaptorV2Calls compound_c_token_v2_calls = 5;
+            // Represents function calls to the AaveATokenV2Adaptor
+            AaveATokenAdaptorV2Calls aave_a_token_v2_calls = 6;
+            // Represents function calls to the AavaDebtTokenV2Adaptor
+            AaveDebtTokenAdaptorV2Calls aave_debt_token_v2_calls = 7;
+            // Represents function calls to the AaveATokenV1Adaptor
+            AaveV3ATokenAdaptorV1Calls aave_v3_a_token_v1_calls = 8;
+            // Represents function calls to the AavaDebtTokenV1Adaptor
+            AaveV3DebtTokenAdaptorV1Calls aave_v3_debt_token_v1_calls = 9;
+            // Represents function calls to the OneInchAdaptorV1
+            OneInchAdaptorV1Calls one_inch_v1_calls = 10;
+            // Represents function calls to the FeesAndReservesAdaptorV1
+            FeesAndReservesAdaptorV1Calls fees_and_reserves_v1_calls = 11;
+            // Represents functionc alls to the ZeroXAdaptorV1
+            ZeroXAdaptorV1Calls zero_x_v1_calls = 12;
+            // Represents function calls to the SwapWithUniswapAdaptorV1
+            SwapWithUniswapAdaptorV1Calls swap_with_uniswap_v1_calls = 13;
+            // Represents function calls to VestingSimpleAdaptor
+            VestingSimpleAdaptorV2Calls vesting_simple_v2_calls = 14;
+            // Represents function calls to the CellarAdaptor
+            CellarAdaptorV1Calls cellar_v1_calls = 15;
+            // Represents function calls to the UniswapV3Adaptor V2
+            UniswapV3AdaptorV2Calls uniswap_v3_v2_calls = 16;
+            // Represents function calls to the AaveV2EnableAssetAsCollatorAdaptor V1
+            AaveV2EnableAssetAsCollateralAdaptorV1Calls aave_v2_enable_asset_as_collateral_v1_calls = 17;
+            // Represents function calls to the FTokenAdaptor V1
+            FTokenAdaptorV1Calls f_token_v1_calls = 18;
+            // Represents function calls to the MorphoAaveV2AToken V1
+            MorphoAaveV2ATokenAdaptorV1Calls morpho_aave_v2_a_token_v1_calls = 19;
+            // Represents function calls to the MorphoAaveV2DebtToken V1
+            MorphoAaveV2DebtTokenAdaptorV1Calls morpho_aave_v2_debt_token_v1_calls = 20;
+            // Represents function calls to the MorphoAaveV3ATokenCollateral V1
+            MorphoAaveV3ATokenCollateralAdaptorV1Calls morpho_aave_v3_a_token_collateral_v1_calls = 21;
+            // Represents function calls to the MorphoAaveV3ATokenP2P V1
+            MorphoAaveV3ATokenP2PAdaptorV1Calls morpho_aave_v3_a_token_p2p_v1_calls = 22;
+            // Represents function calls to the MorphoAaveV3DebtToken V1
+            MorphoAaveV3DebtTokenAdaptorV1Calls morpho_aave_v3_debt_token_v1_calls = 23;
+            // Represents function calls to the BalancerPoolAdaptor V1
+            BalancerPoolAdaptorV1Calls balancer_pool_v1_calls = 24;
```

Comment:
BalancerPoolAdaptorV1Calls can be removed as we are not allowed to call it in the Flashloan due to reentrancy

Reply:
this would be true for aave v3's flash loan adaptor calls too right? remove its own calls from the set?